proc-macro-helpers/src/lib.rs

```diff
@@ -16,9 +16,8 @@ extern crate quote;
 extern crate syn;
 
 use proc_macro::TokenStream;
-use proc_macro2::TokenStream as TokenStream2;
+use proc_macro2::{Span, TokenStream as TokenStream2};
 use quote::ToTokens;
-use quote::__rt::Span;
```

Comment:
Oh wow, I can't believe we were importing from here without even a comment to justify why! 😬

Reply:
+1 Thanks for fixing this!